client/allocdir/alloc_dir_nonlinux_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+//go:build !linux
+// +build !linux
```

Comment:
BSDs are gonna be sad, but we don't officially support them anyway. Perhaps:
```suggestion
//go:build darwin || windows
```

(can we drop the old `// +build` way? it was replaced 3+ years ago)